app/src/main/res/xml/customization_preferences.xml

```diff
@@ -7,7 +7,6 @@
     <androidx.preference.PreferenceCategory
         android:layout="@layout/preference_cat_style"
         android:title="@string/preferences_theme"
-        app:allowDividerBelow="true"
```

Comment:
I think these dividers were just added in https://github.com/mozilla-mobile/fenix/pull/15917. I am not quite sure we should be removing them.

Reply:
I added `app:allowDividerAbove="true"` in `preference_cat_style`, I removed this here to avoid redundancy. The dividers are still there.